src/components/autocomplete/demoRepeatMode/index.html

```diff
@@ -0,0 +1,97 @@
+              md-item-text="item.name"
+              md-min-length="0"
+              input-aria-describedby="standard-autocomplete-description"
+              input-aria-label="Current Repository"
+              placeholder="Pick an Angular repository"
+              md-menu-class="autocomplete-standard-template"
+              md-menu-container-class="standard-container"
+              md-mode="standard">
+            <md-item-template>
+              <span class="item-title">
+                <md-icon md-svg-icon="img/icons/octicon-repo.svg"></md-icon>
+                <span> {{item.name}} </span>
+              </span>
+              <p class="item-desc">{{item.desc}}</p>
+              <span class="item-metadata">
+                <span>
+                  <strong>{{item.watchers}}</strong> watchers
+                </span>
+                <span>
+                  <strong>{{item.forks}}</strong> forks
+                </span>
+              </span>
+            </md-item-template>
+          </md-autocomplete>
+        </form>
+      </div>
+
+      <div class="instance-wrapper" flex-gt-xs="50" ng-controller="DemoCtrl as ctrl">
+        <form ng-submit="$event.preventDefault()">
+          <p id="virtual-autocomplete-description">
+            Omit the <code>md-mode</code> attribute (or specify <code>virtual</code>) to
+            use <code>md-virtual-repeat</code> for autocomplete entries. Virtual repeat
+            will provide an angular-managed endless scrolling list, with related performance
+            optimizations. Note that all entries in <code>md-virtual-repeat</code> list
+            must have the same height for the endless-lists to work properly.
+          </p>
+          <h4>Virtual mode</h4>
+          <md-autocomplete
+              id="virtual-autocomplete"
+              ng-disabled="ctrl.isDisabled"
+              md-no-cache="ctrl.noCache"
+              md-selected-item="ctrl.selectedItem"
+              md-search-text-change="ctrl.searchTextChange(ctrl.searchText)"
+              md-search-text="ctrl.searchText"
+              md-selected-item-change="ctrl.selectedItemChange(item)"
+              md-items="item in ctrl.querySearch(ctrl.searchText)"
+              md-item-text="item.name"
+              md-min-length="0"
+              input-aria-describedby="virtual-autocomplete-description"
+              input-aria-label="Current Repository"
+              placeholder="Pick an Angular repository"
+              md-menu-class="autocomplete-virtual-template"
+              md-menu-container-class="virtual-container"
+              md-mode="virtual">
+            <md-item-template>
+              <span class="item-title">
+                <md-icon md-svg-icon="img/icons/octicon-repo.svg"></md-icon>
+                <span> {{item.name}} </span>
+              </span>
+              <p class="item-desc">{{item.desc | limitTo:40}}{{item.desc.length > 40 ? '...': ''}}</p>
```

Comment:
Let's split this into multiple lines so that the code doesn't wrap in the demo code block:
```html
<p class="item-desc">
  {{item.desc | limitTo:40}}{{item.desc.length > 40 ? '...': ''}}
</p>
```